praw/objector.py

```diff
@@ -21,18 +21,14 @@ def parse_error(cls, data):
             # assume it's a possibility here.
             return None
 
-        errors = data.get("json", {}).get("errors")
-        if errors is None:
+        errors = data.get("json", {}).get("errors", ())
+        if not errors:
             return None
-        if len(errors) < 1:
-            # See `Collection._fetch()`.
-            raise ClientException("successful error response", data)
         if len(errors) > 1:
```

Comment:
Can we change this to `assert len(errors) == 1`?

Reply:
Just leave it like this for now as I have plans for this code path next PR.